source/ncconvert/utility/EnumExtensions.cpp

```diff
@@ -7,6 +7,31 @@
 
 namespace nc::convert
 {
+auto CanOutputMany(asset::AssetType type) -> bool
+{
+    switch(type)
+    {
+        case asset::AssetType::AudioClip:
+            return false;
+        case asset::AssetType::CubeMap:
+            return false;
+        case asset::AssetType::ConcaveCollider:
+            return false;
+        case asset::AssetType::HullCollider:
+            return false;
+        case asset::AssetType::Mesh:
+            return true;
+        case asset::AssetType::Texture:
+            return false;
+        default:
```

Comment:
Was the compiler complaining about no default? IIRC, if you leave the default out, you can get a warning if you add a new enum value and forget to add it to the switch (from gcc maybe?).

Reply:
Ahh `ToString` below has default too.